test/integration/test_create_dss_docker_image.py

```diff
@@ -0,0 +1,68 @@
+import docker
+import logging
+import pytest
+import requests
+import time
+
+from exasol.ds.sandbox.lib.dss_docker import DssDockerImage
+from datetime import datetime
+
+
+@pytest.fixture(scope="session")
+def dss_docker_container():
+    timestamp = f'{datetime.now().timestamp():.0f}'
+    testee = DssDockerImage(
+        "my-repo/dss-test-image",
+        version=f"{timestamp}",
+        publish=False,
+        log_level=logging.INFO,
+    )
+    print(
+        "\n- Using"
+        f' Docker container {testee.container_name}'
+        f' with image {testee.image_name}'
+    )
+    testee.create()
+    client = docker.from_env()
+    mapped_ports = {'8888/tcp': 8888}
+    container = client.containers.create(
```

Comment:
You don't map the docker socket. I think, we need another cli/API for starting a container correctly. However, this is a new ticket.